File: Projekat/Models/Usluga.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore.Infrastructure;
namespace Models
{
    [Table("Usluga")]
    public class Usluga
    {
        [Key]
        public int IDUsluge{get;set;}

        [Required]
        public string VrstaUsluge{get;set;}

        [Required]
        public int Cena{get;set;}

        

        //public virtual Kozmeticar Kozmeticari{get; set;}

        public List<Kozmeticar> Kozmeticari{get; set;}

    }
}

File: Projekat/Client/Usluga.js
import {Kozmeticar} from "./Kozmeticar.js";
export class Usluga{
    constructor(idUsluge,vrstaUsluge,cena)
    {
        this.idUsluge=idUsluge;
        this.vrstaUsluge=vrstaUsluge;
        this.cena=cena;
        this.Kozmeticari=[];
    }
    
    DodajKozmeticara(kozmeticar)
    {
        if(this.IDUsluge==kozmeticar.uslugaid)
        {
            this.Kozmeticari.push(kozmeticar);
        }
    }
    
    
}

